tests/test_registration_embargoes.py

```diff
@@ -8,7 +8,7 @@
 from tests.base import fake, OsfTestCase
 from tests.factories import (
     AuthUserFactory, EmbargoFactory, NodeFactory, ProjectFactory,
-    RegistrationFactory, UserFactory, UnconfirmedUserFactory
+    RegistrationFactory, UserFactory, UnconfirmedUserFactory, DraftRegistrationFactory
```

Comment:
In this case I've recently found a number of scripts that import (and import from) the same module 2-3 times in the same script in different ways, often under different names.

I agree that this is a style opinion, but it's one based on experience and recent observations. If you have an alternate format that is consistent, predictable, and organized, I'm always open to reasonable alternatives. 

Just as long as we don't mix usages of BareFactory, factories.BareFactory, and aliased_factories.BareFactory in the same module again... I think that the end goal is what matters here.